app/assets/stylesheets/application.scss

```diff
@@ -14,6 +14,24 @@ $govuk-compatibility-govuktemplate: true;
 @import 'govuk_publishing_components/components/select';
 @import 'govuk_publishing_components/components/step-by-step-nav-header';
 
+// Map a few legacy variables to `govuk-frontend` equivalents
+// to avoid introducing a dependency on govuk_frontend_toolkit
+$text-colour: $govuk-text-colour;
+$secondary-text-colour: $govuk-secondary-text-colour;
+$link-visited-colour: $govuk-link-visited-colour;
```

Comment:
It looks like this is only used in one place (`publisher.scss`) so could this line be removed and use `govuk-link-visited-colour` directly there instead?

Reply:
Same with `link-hover-colour`, `link-colour` and `canvas-colour`.